packages/client/src/App.test.tsx

```diff
@@ -1,12 +1,5 @@
-import { render } from '@testing-library/react';
-import App from './App';
+export {}
```

Comment:
выглядит как костыль, похоже надо поднастроить jest

Reply:
Это нужно: 'App.test.tsx' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.

Reply:
ок, тогда это хотя бы в конец файла надо поместить